src/main/sass/theme.scss

```diff
@@ -0,0 +1,1341 @@
+
+$color-white: #FFF;
```

Comment:
Yup. They aren't in the style.scss, but I'm not sure it matters. If it does, we can fix it then :D